extension/llm/runner/constants.h

```diff
@@ -21,7 +21,8 @@ inline constexpr auto kUseSDPAWithKVCache = "use_sdpa_with_kv_cache";
 
 // Multimodal method name conventions
 inline constexpr auto kImageEncoderMethod = "image_encoder";
```

Comment:
are these documented aot anywhere?

Reply:
General follow-up issue for better documentation, including this one - https://github.com/huggingface/optimum-executorch/issues/129